bot/exts/info/pypi.py

```diff
@@ -1,14 +1,16 @@
+import contextlib
```

Comment:
Could be good to be consistent, everywhere else it's `from contextlib import suppress`.

Reply:
Wasn't sure if it would change the import orders (I don't really understand how the linting works for imports)

Reply:
[Committed](https://github.com/python-discord/bot/pull/1696/commits/d52bf9d921465320ba8f70fb36933aed8340480d) with no issues